src/Core/Geographic/Extent.js

```diff
@@ -47,8 +47,7 @@ cardinals.push(new Coordinates('EPSG:4326', 0, 0, 0, 0));
 const _c = new Coordinates('EPSG:4326', 180, 85.06);
 _c.as('EPSG:3857', _c);
 // Get bound dimension in 'EPSG:3857'
-const sizeX = _c._values[0] * 2;
-const sizeY = _c._values[1] * 2;
+export const worldDimension3857 = { x: _c._values[0] * 2, y: _c._values[1] * 2 };
```

Comment:
Not a big fan of this, this needs to be removed once `Coordinates` has been refactored with `x`, `y` etc... getter/setter.

Reply:
I will change that.